### PR TITLE
Fix new swiftlint warnings

### PR DESCRIPTION
**Pull request by vargaat**
refs: MBL-16197
affects: Parent, Student, Teacher
release note: none

test plan:
- PR checks/tests should pass

### CODE REVIEW

**Comment by inst-danger** [comment]
<details><summary>Parent Build QR Code:</summary>

</details>

**Comment by inst-danger** [comment]
<details><summary>Teacher Build QR Code:</summary>

</details>

**Comment by inst-danger** [comment]
<details><summary>Student Build QR Code:</summary>

</details>

File: Core/Core/Files/Model/FileSubmissionsStatus.swift [hunk 1]
```diff
@@ -46,12 +46,7 @@ class FileSubmissionsStatus {
         var result = false
 
         accessQueue.sync {
-            for fileID in fileIDs {
-                if inProgressSubmissions.contains(fileID) {
-                    result = true
-                    break
-                }
-            }
+            result = fileIDs.contains { inProgressSubmissions.contains($0) }
```
Comment (balintbartok) [inline Core/Core/Files/Model/FileSubmissionsStatus.swift:49]: nice

**Review by balintbartok — APPROVED** [review]

**Review by davidvarga93 — APPROVED** [review]
QA + 1

**Pull request merged**